c/CMakeLists.txt

```diff
@@ -2,7 +2,20 @@
 # Define libraries
 #*******************************************************************************
 
+# Memory management system
 add_library(apple_memory INTERFACE)
 
 target_include_directories(apple_memory
     INTERFACE ${CMAKE_CURRENT_SOURCE_DIR})
+
+# Logging system
+add_library(apple_log INTERFACE)
+
+target_include_directories(apple_log
+    INTERFACE ${CMAKE_CURRENT_SOURCE_DIR})
+
+target_compile_definitions(apple_log
+    INTERFACE LOG_CONFIG=${LOG_CONFIG})
+
+target_compile_options(apple_log
```

Comment:
at some point I will need to align `CMAKE_BUILD_TYPE` cache variable to this `-g`